src/db.rs

```diff
@@ -651,7 +654,7 @@ mod tests {
             .await
             .unwrap()
             .unwrap();
-        let sst_format = SsTableFormat::new(4096, 10, None);
+        let sst_format = SsTableFormatBuilder::new().with_min_filter_keys(10).build();
```

Comment:
is `with_min_filter_keys` needed if it's 10? Noticed the change above didn't do this.

_EDIT: Hmm, I think the one above is wrong, actually. Looks like min filter keys defaults to 0 now in the builder._